verify/vm-prep

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
```

Comment:
This and the above XML are by and large a copy of cockpit's files. Why can't we just use these?